salt/modules/glusterfs.py

```diff
@@ -660,3 +664,82 @@ def list_quota_volume(name):
         ret[path] = _etree_to_dict(limit)
 
     return ret
+
+
+def get_op_version(name):
+    '''
+    Returns the glusterfs volume op-version
```

Comment:
Can you add a `.. versionadded:: Fluorine` tag here for this new function?

This comment applies to the other functions you've added here as well. :)